templates/microcloud/index.html

```diff
@@ -0,0 +1,444 @@
+{% extends 'base_index.html' %}
+
+{% block title %}MicroCloud &ndash; low-touch open source cloud solution{% endblock %}
+
+{% block meta_description %}MicroClouds offer a lightweight and automated solution for edge clouds and small scale private clouds. Best of open source cloud infrastructure components integrated seamlessly.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/17iBbFCINLYjPBOYWpgoq9SreyYNkR0Sb5jgx5iTo6MA/edit{% endblock meta_copydoc %}
+
+{% block body_class %}is-paper{% endblock body_class %}
+
+{% block content %}
+
+<section class="p-strip--whitesuru is-shallow u-no-padding--bottom">
+  <div class="p-section">
+    <div class="row">
+      <div class="col-start-large-4 col-9">
+        <div class="p-section--shallow">
+          <h1>The power of cloud<br/>anywhere you need it</h1>
+        </div>
+        <div class="p-section--shallow">
+          <p>Straightforward and lightweight, but featureful.<br/>MicroClouds serve compute near consumers, wherever they may be.</p>
+        </div>
+        <hr>
+        <a class="p-button--positive">Get in touch</a>
+        <a href="https://ubuntu.com/engage/guide-to-micro-clouds">Read our guide to MicroCloud</a>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--25-75">
+    <hr class="p-rule"/>
+    <div class="col">
+      <h2>What is a MicroCloud?</h2>
+    </div>
+    <div class="col">
+      <div class="row">
+        <div class="col-6">
+          <div class="u-embedded-media">
+            <iframe class="u-embedded-media__element" title="What is a MicroCloud?" width="560" height="315" src="https://www.youtube.com/embed/FnzUcGg60Qk?si=CdadVAcSZGLHLi6S" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="u-fixed-width">
+    <hr class="p-rule"/>
+    <h2>Deploy a fully functional cloud in minutes:</h2>
```

Comment:
I wonder if we should hide this from screen readers. Everything gets read correctly, but without visual queues is there much value in it? What do you think @juanruitina?

Reply:
Very cool effect. If we are not hiding it, can we make tweaks to the responsive smaller scree sizes? I think the text needs to be smaller in smaller screen sizes, the cursor is not aligning correctly. But not a block in my opinion or we can just hide like Pete says

![Screenshot from 2023-11-01 09-58-10](https://github.com/canonical/canonical.com/assets/14939793/a3d9dfa2-3dda-41d9-a458-489e2d3e217e)

Reply:
I added an aria-hidden to is as it doesn't read both commands, only the one currently being typed. I'll make a follow-up for design on smaller screens though!